src/main/kotlin/no/nav/yrkesskade/saksbehandling/graphql/common/model/Journalfoeringdetaljer.kt

```diff
@@ -0,0 +1,5 @@
+package no.nav.yrkesskade.saksbehandling.graphql.common.model
+
+data class Journalfoeringdetaljer(
+    val avsender: String
```

Comment:
Er dette ment som data som skal oppdateres på journalpost etterhvert? Vet ikke om det er relevant i denne sammenheng, men i så fall opererer den datamodellen med litt mer info, for eksempel om det er en person eller organisasjon. 
Se feltet `avsenderMottaker` på https://confluence.adeo.no/display/BOA/opprettJournalpost

Reply:
Godt poeng! Jeg har egentlig bare laget støtte for frontend feltet. Jeg antar at modellen kommer til å endres etterhvert som vi går.